### PR TITLE
Update sql-machine-learning-services-windows-install.md

### PR DESCRIPTION
**Pull request by joshts0**
Adding clarification that the feature cannot be installed in an existing FCI and must be installed during installation time.

### CODE REVIEW

**Comment by ShannonLeavitt** [comment]
@WilliamDAssafMSFT 

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/machine-learning/install/sql-machine-learning-services-windows-install.md [hunk 1]
```diff
@@ -21,11 +21,11 @@ This article shows you how to install [SQL Server Machine Learning Services](../
 
 ## <a name="bkmk_prereqs"> </a> Pre-installation checklist
 
-+ A database engine instance is required. You can't install just Python or R features, although you can add them incrementally to an existing instance.
++ A database engine instance is required. You can't install just Python or R features, although you can add them incrementally to an existing **standalone** instance.
```
Comment (rwestMSFT) [inline docs/machine-learning/install/sql-machine-learning-services-windows-install.md]:
```suggestion
+ A database engine instance is required. You can't install just Python or R features, although you can add them incrementally to an existing standalone instance.
```

File: docs/machine-learning/install/sql-machine-learning-services-windows-install.md [hunk 2]
```diff
@@ -21,11 +21,11 @@ This article shows you how to install [SQL Server Machine Learning Services](../
...
 
 + For business continuity, [Always On availability groups](../../database-engine/availability-groups/windows/overview-of-always-on-availability-groups-sql-server.md) are supported for Machine Learning Services. Install Machine Learning Services, and configure packages, on each node.
 
-+ Installing Machine Learning Services *is not supported* on an [Always On failover cluster instance](../../sql-server/failover-clusters/windows/always-on-failover-cluster-instances-sql-server.md) in SQL Server 2017. It's supported with SQL Server 2019 and later.
++ Installing Machine Learning Services *is not supported* on an [Always On failover cluster instance](../../sql-server/failover-clusters/windows/always-on-failover-cluster-instances-sql-server.md) in SQL Server 2017. It's supported with SQL Server 2019 and later. You must install Machine Learning Services at installation time, and features cannot be added to an existing failover instance once installed.
```
Comment (rwestMSFT) [inline docs/machine-learning/install/sql-machine-learning-services-windows-install.md]:
```suggestion
+ Installing Machine Learning Services *is not supported* on an [Always On failover cluster instance](../../sql-server/failover-clusters/windows/always-on-failover-cluster-instances-sql-server.md) in SQL Server 2017. It's supported with SQL Server 2019 and later. You must install Machine Learning Services at installation time, and features can't be added to an existing failover cluster instance once installed.
```

**Comment by WilliamDAssafMSFT** [comment]
Thanks @joshts0 for taking the time to submit this PR! Good catch. Your commit will be merged soon. After the updated article is published, GitHub will list you as an article contributor. 

Thanks again for this feedback, your suggestions help improve our documentation.

**Comment by WilliamDAssafMSFT** [comment]
#sign-off

**Comment by Jak-MS** [comment]
merged

**Pull request merged**